gcp/kbn-dev.sh

```diff
@@ -89,15 +93,21 @@ case $ACTION in
       -var="gcp_name=${GCP_NAME}" \
       -var="kibana_repo_url=${REPO_URL}" \
       -var="kibana_repo_branch=${BRANCH}" \
+      -var="kibana_server_password=${PASSWORD}" \
       -auto-approve
     terraform -chdir="${SCRIPT_DIR}" workspace select "${WORKSPACE_NAME}"
     PUBLIC_IP=$(terraform -chdir="${SCRIPT_DIR}" output -json | jq  -r '.public_ip.value')
     KIBANA_URL=$(terraform -chdir="${SCRIPT_DIR}" output -json | jq  -r '.kibana_url.value')
+    KIBANA_SERVER_PASSWORD=$(terraform -chdir="${SCRIPT_DIR}" output -json | jq  -r '.kibana_elastic_password.value')
+    log "🥬 Password ${KIBANA_SERVER_PASSWORD}"
```

Comment:
Maybe we should avoid logging the password here, or perhaps only log it if it is using a generated password?

Reply:
that's a good idea!

Reply:
done